src/services/videoconference/index.js

```diff
@@ -43,319 +43,314 @@ const { ObjectId } = require('../../helper/compare');
 
 const CLIENT_HOST = Configuration.get('HOST');
 
-class VideoconferenceBaseService {
-	constructor(app) {
-		this.app = app;
-		this.docs = {};
+function throwOnValidationErrors(scopeId, scopeName, options = null) {
+	if (!idAndScopeNameAreValid({ scopeId, scopeName })) {
+		throw new BadRequest('id or scopeName invalid');
 	}
-
-
-	static throwOnValidationErrors(scopeId, scopeName, options = null) {
-		if (!VideoconferenceBaseService.idAndScopeNameAreValid({ scopeId, scopeName })) {
-			throw new BadRequest('id or scopeName invalid');
-		}
-		if (options !== null) {
-			const validOptions = VideoconferenceBaseService.getValidOptions(options);
-			if (!lodash.isEqual(options, validOptions)) {
-				throw new BadRequest('options invalid'); // todo cleanup
-			}
+	if (options !== null) {
+		const validOptions = getValidOptions(options);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['getValidOptions' was used before it was defined.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5401970)